Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,28 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class FileLoggerTests
     {
-        
+        [TestMethod]
+        public void Log_GivenLog_Success()
```

Comment:
You've got the naming convention correct, but we should be more explicit with our expected result. `Success` is a very general idea, and could be up for interpretation. 
```suggestion
        public void Logger_Created_AppendsMessageToFile()
```